### ISSUE TITLE
[Docs] Add more FAQs about the chocie of parameters

### ISSUE THREAD

**Issue by njzjz** [deepmodeling/deepmd-kit#2268]
### Summary

Currently, we have a benchmark of network size in the [FAQ](https://docs.deepmodeling.com/projects/deepmd/en/latest/troubleshooting/howtoset_netsize.html). I think we should choose some typical datasets to test more parameters and benchmark their accuracy and speed (for both CPUs and GPUs).

### Detailed Description

I have several ideas here:

- [ ] `type_one_side`: #2265
- [ ] `precision`: In some (or most) systems, FP64 does not improve accuracy compared to FP32.
- [ ] `activation_function`
- [ ] the type of the descriptor: `se_e2_a`, `se_e2_r`, `se_e3`, `local_frame`
- [ ] training steps
- [ ] loss function, i.e., the weight of energy/forces
- [ ] learning rate
- [ ] batch size

**Comment by wangsl613** [comment]
Given that FP64 does not enhance accuracy over FP32, how can I switch from FP64 to FP32 in deepmd during training?
Please let me know.
Many thanks!
